inspire_dojson/hep/rules/bd5xx.py

```diff
@@ -53,16 +64,29 @@ def public_notes(self, key, value):
             match = IS_DEFENSE_DATE.match(public_note)
             if match:
                 thesis_info['defense_date'] = match.group('defense_date')
+            elif _means_not_curated(public_note):
+                curated = False
             else:
                 public_notes.append({
                     'source': source,
                     'value': public_note,
                 })
 
+    self['curated'] = curated
     self['thesis_info'] = thesis_info
     return public_notes
 
 
+@hep2marc.over('500', '^curated$')
+def curated2marc(self, key, value):
+    result = self.get('500', [])
+
+    if value is False:
+        result.insert(0, {'a': '* Brief entry *'})
```

Comment:
this is wrong: core -> Temporary, non-core -> Brief.

Reply:
Oh, so "Temporary" and "Brief" mean different things, and I need to know if the record is also CORE to know which to choose?

Reply:
yes, core records are curated eventually and lose their temporary flag. non-core records on the other hand are generally not curated, so their state is not temporary.